consensus/istanbul/backend/announce.go

```diff
@@ -50,6 +49,8 @@ const (
 // QueryEnodeGossipFrequencyState specifies how frequently to gossip query enode messages
 type QueryEnodeGossipFrequencyState int
 
+type announceOpFunc func(uint)
```

Comment:
Could you add a comment and/or move this closer to where it is used?

Reply:
It's not needed anymore and it's removed.